kosat-core/src/jvmTest/kotlin/org/kosat/DiamondTests.kt

```diff
@@ -2,189 +2,215 @@ package org.kosat
 
 import com.github.lipen.satlib.solver.MiniSatSolver
 import korlibs.time.measureTimeWithResult
+import korlibs.time.roundMilliseconds
+import okio.FileSystem
+import okio.Path.Companion.toOkioPath
+import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.TestInstance
 import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
 import org.junit.jupiter.params.provider.MethodSource
+import org.kosat.cnf.CNF
+import org.kosat.cnf.from
 import java.io.File
 import java.nio.file.Files
+import java.nio.file.Path
 import java.nio.file.Paths
+import java.time.LocalDateTime
+import kotlin.io.path.extension
+import kotlin.io.path.isRegularFile
+import kotlin.io.path.relativeTo
 import kotlin.math.abs
 import kotlin.math.sign
 import kotlin.random.Random
 import kotlin.streams.toList
 import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+// Lazy messages are not in JUnit
```

Comment:
What about that? https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html#assertTrue-java.util.function.BooleanSupplier-java.util.function.Supplier-

We are using JUnit5 anyways, for parametrized tests, so you can use assertions from it, not just from kotlin.test